docs/rules/no-magic-number.md

```diff
@@ -0,0 +1,50 @@
+# Disallow Magic Numbers (no-magic-number)
+
+'Magic numbers' are numbers that occur multiple time in code without an explicit meaning.
+They should preferably be replaced by named constants.
+
+```js
+if (foo == null) {
+  bar();
+}
+```
```

Comment:
If you make your change, then do `git commit -a --amend --no-edit`, that'll add the change to the current commit. You can then `git push -f` to force push to your `new-rule-no-magic-number` branch, which will automatically update this PR.